lambda/postSchedulePeerTeachers/Scheduler/src/main/java/scheduler/Handler.java

```diff
@@ -58,13 +57,14 @@ public APIGatewayProxyResponseEvent handleRequest(final APIGatewayProxyRequestEv
         APIGatewayProxyResponseEvent response = new APIGatewayProxyResponseEvent()
                 .withHeaders(headers);
 
-        GetParameterRequest parameterRequest = GetParameterRequest.builder()
-                .name("/tias/prod/db-password")
-                .withDecryption(true)
-                .build();
-        SsmClient ssmclient = SsmClient.create();
-        GetParameterResponse parameterResult = ssmclient.getParameter(parameterRequest);
-        String dbpassword = parameterResult.parameter().value();
+        // GetParameterRequest parameterRequest = GetParameterRequest.builder()
+        //         .name("/tias/prod/db-password")
+        //         .withDecryption(true)
+        //         .build();
+        // SsmClient ssmclient = SsmClient.create();
+        // GetParameterResponse parameterResult = ssmclient.getParameter(parameterRequest);
+        // String dbpassword = parameterResult.parameter().value();
+        String dbpassword = "admin";
```

Comment:
Does this need to be reverted back so it works when you upload it? It looks like this might be for testing?